### DISCUSSION TITLE
🎉 Commonhaus Foundation: Now Open!

### DISCUSSION THREAD

**Discussion by ebullient** [commonhaus/foundation#123] [Announcements]
We are excited to formally announce the formation of the Commonhaus Foundation!

Below, you'll find the full text of our announcement [press release](https://docs.google.com/document/d/18KiwTIqkGKF7k_DF0oPdC9wGDVSNDvhmBbZUncwSfnw/edit).

**Our first priority**: members! We'd like to enter the next stage of building this organization, and to do that, we need members. 

**Have a project in mind?** [Create a discussion](https://github.com/commonhaus/foundation/discussions/categories/joining-commonhaus) or contact us at 

Please check out our [community page](https://www.commonhaus.org/community/), and build with us! 

---

### Commonhaus Foundation Debuts to Provide Succession Planning and Fiscal Support for Self-governing Open Source Projects

Windermere, Florida - All remote – April 9, 2024 –  Today, the [Commonhaus Foundation](https://www.commonhaus.org/) launches as a new model for established open source libraries and frameworks seeking a neutral home. The non-profit’s mission is to ensure a place where open source projects can continue to thrive for future generations. Commonhaus is prioritizing succession planning, minimal governance, and financial assistance for its member projects. 

This initiative emerges in response to the dynamic needs of open source communities seeking foundation frameworks that support growth and transition without imposed constraints. By focusing on succession planning, Commonhaus ensures that open source projects can navigate the challenges of leadership and contributor changes with ease, maintaining their influence and functionality across generations of technology evolution.

“The open source ecosystem has evolved significantly in recent years, with existing foundations laying the groundwork for many successful projects,” said [Erin Schnabel](https://www.linkedin.com/in/erinschnabel/), a Commonhaus Foundation founder and the organization’s chairperson. “Yet, with the rise of fiscal hosting platforms and self-governance models on sites like Open Collective and GitHub, there's a clear need for a different approach. Traditional governance models may not align with the needs of projects that have cultivated their own practices and infrastructure. Such projects often find the prospect of merging into existing foundations to be challenging, costly, or unattractive, leaving key needs unaddressed. These include the need for a neutral, lasting repository for their assets and access to funds, especially for those without vendor backing.”

The Commonhaus Foundation plans to fill this gap to ensure that projects of all governance and operational models have the support needed to thrive. Commonhaus is dedicated to extending the life and success of projects, filling a critical need for those that do not fit within the existing frameworks yet play a vital role in the open source community.

[Trisha Gee](https://www.linkedin.com/in/trishagee/), software engineer, Java Champion and author, stated: “I’m excited to hear about Commonhaus, it very nicely fills an important gap in support for open source projects. The focus on community and continuity is crucial for their success, ensuring sustainability and preventing maintainer burnout. It’s not unusual for these projects to be used in software that has a significant impact on people’s lives around the globe, and it’s important the maintainers get the support they need to sustain the project. Open source projects that are successful and self-organised shouldn’t have to change their ways of working to have the support of a foundation, and they deserve to rely on a foundation to ensure the longevity of the project. Commonhaus is going to be a new model in sustainable innovation, focused on the power of collective effort.”

Open source projects joining Commonhaus at its launch include [Hibernate](https://hibernate.org/), [Jackson](https://github.com/FasterXML/jackson), [OpenRewrite](https://github.com/openrewrite), [JBang](https://www.jbang.dev/), [JReleaser](https://jreleaser.org/), and [Morphia](https://morphia.dev/).

The foundation is calling on developers, contributors, projects and sponsors to join its cause. It promises a future in which open source projects can continue to grow and serve the global technology community without fear of obsolescence.

Commonhaus Foundation membership and support can be initiated through [sponsorship (GitHub)](https://github.com/sponsors/commonhaus). Becoming a Commonhaus sponsor financially supports the organization. It also enables community members to hold foundation offices, vote in foundation elections, and influence the strategic direction of sustainable development of open source projects.

To learn more about the Commonhaus Foundation initiatives, visit [www.commonhaus.org](https://www.commonhaus.org/), follow it on [LinkedIn](https://www.linkedin.com/company/commonhaus-foundation/about/), or contact the organization at [hello@commonhaus.org](mailto:hello@commonhaus.org).

Members of the Commonhaus Foundation will be present at several industry conferences from April 9-11. Catch [Erin Schnabel](https://www.linkedin.com/in/erinschnabel/), [Ken Finnigan](https://www.linkedin.com/in/kenfinnigan/), [Justin Lee](https://www.linkedin.com/in/justinlee/), [Olga Kundzich](https://www.linkedin.com/in/olgakundzich/), and [Jonathan Schneider](https://www.linkedin.com/in/jonkschneider/) at Devnexus in Atlanta; [Andres Almiray](https://www.linkedin.com/in/aalmiray/) at JavaLand in Nürburgring; and [Cesar Saavedra](https://www.linkedin.com/in/saavedracesar/) at Google Cloud Next in Las Vegas. They're ready to talk about the foundation's work, field any queries you might have, and explore ways we can work together to advance open source development.

### About Commonhaus Foundation

The Commonhaus Foundation is a non-profit organization dedicated to the long-term sustainability of open source projects. By providing a neutral home, developer-led governance, and fiscal sponsorship, the foundation ensures that essential libraries and frameworks continue to advance, remaining accessible for future generations of developers and businesses across the spectrum.

#### Red Hat, stewards of [Hibernate](https://hibernate.org/)

[Gavin King](https://www.linkedin.com/in/gavinking/), representative for Hibernate, explains the reasoning behind the move to Commonhaus: “Hibernate is already ubiquitous in the Java world, and has seen an explosion of activity since the release of version 6. Commonhaus Foundation is the “neutral ground” we’ve chosen to continue growing our community and to seek closer engagement with other organizations doing interesting things with data. Commonhaus provides support and continuity for self-governing projects, making it a perfect host for projects which are already well-run. Red Hat's leadership and sponsorship continue to be a source of stability and user confidence, and we now hope to open up more space for other groups and companies to feel included in the project.”

#### Moderne, stewards of [OpenRewrite](https://github.com/openrewrite)

[Jonathan Schneider](https://www.linkedin.com/in/jonkschneider/), CEO and co-founder of Moderne and creator of the OpenRewrite project, expressed support for the foundation: "Joining Commonhaus aligns perfectly with the long-term vision and mission of OpenRewrite. Our project is dedicated to delivering automated, open source code migrations, upgrades and security fixes swiftly, leveraging the collective effort of the entire community. We've always believed that managing dependencies requires a broad coalition; it's a task too vast for any single entity. Moderne is committed to the principles of pure open source, and we recognize that when an entire community—including large enterprises—depends on your projects, providing solid assurances becomes paramount. Commonhaus offers these assurances through succession planning beyond a single vendor. It supports our commitment to keep OpenRewrite vibrant, community-driven, and accessible, ensuring its utility and relevance for years to come."

#### [JBang](https://www.jbang.dev/)

[Max Rydahl Andersen](https://www.linkedin.com/in/maxrydahlandersen/), the creator of JBang, shares his insights on joining the CommonHaus foundation: “JBang is making Java more accessible, simplifying the learning and experimentation process for both students and professionals. My journey with JBang began as a personal project to renew my Java skills, which led me to (again) appreciate the vital role of contributors to java libraries in keeping the Java ecosystem thriving. While I explored various foundations for JBang, none seemed to align with our vision until Commonhaus. With its commitment to ‘taking care of the basics’ and allowing 'hands-off project execution', Commonhaus stood out as the ideal home for JBang. I'm excited for JBang to be part of this new chapter in Java's adventure, ensuring its continued growth and impact.”

#### [JReleaser](https://jreleaser.org/)

[Andres Almiray](https://www.linkedin.com/in/aalmiray/), creator and current lead of the JReleaser project, says the following regarding the foundation: "JReleaser has steadily grown in popularity since its inception back in 2021. The project simplifies and automates releases for all kinds of projects in such a way that consumers are only required to configure what they need, the tool leverages convention over configuration and sensible defaults to perform its job. We knew it would be beneficial for everyone if the project were to move to a foundation at some point in its lifetime. Commonhaus provides everything the project needs at its current state while also offering assurances for its future. The timing to join Commonhaus couldn't be better as JReleaser will celebrate one more anniversary of its first release by the time the foundation is officially launched".

#### [Jackson](https://github.com/FasterXML/jackson)

[Tatu Saloranta](https://www.linkedin.com/in/tatu-saloranta-b2b36/), the creator and project lead of Jackson, expresses his enthusiasm for joining the Commonhaus Foundation: "While Jackson components are widely used within the Java software ecosystem, our core project team remains compact, and our processes minimal. Over the years, we have occasionally considered the desirability and feasibility of affiliating with well-known open source foundations. However, none seemed to quite align with the preferences of our project and team. After observing the formation of the Commonhaus Foundation, we are confident that we have found our future home. The foundation's goals align closely with ours, and it emphasizes enabling individual projects to maintain focus on their core competency: the development of high-quality open source software."

#### [Morphia](https://morphia.dev/)

[Justin Lee](https://www.linkedin.com/in/justinlee/), the creator of Morphia, adds the following: "Morphia has had a long, complicated history leading to its current situation. Like other projects, various foundations and communal “homes” have been considered but none have really felt like a great fit.  Commonhaus’s goals align with Morphia’s for a long-term stable home that isn’t dependent on one person or corporate entity. By securing a long-term home and lightweight guidance in governance for the project, we can focus more on the project itself rather than the logistics surrounding its maintenance.  By joining the Commonhaus Foundation, we can begin to build and share common infrastructure with our hausmates and make life better for all of us."

**Comment by pragnesh** [comment]
Morphia project link is incorrect

**Reply by ebullient** [reply]
Good catch... fixed. =)

**Comment by rainboyan** [comment]
First of all, congratulations on the establishment of the Foundation, which is a major change and innovation for open source projects such as Hibernate, I am also currently maintaining an open source project, very interested in how to join the Foundation as an open source project, where there is information about the relevant.

**Reply by ebullient** [reply]
We are still finalizing that process. We have a lot of work to do. Please create a new discussion in the [joining commonhaus](https://github.com/commonhaus/foundation/discussions/categories/joining-commonhaus) category.

**Reply by rainboyan** [reply]
@ebullient ok, thank you for your reply.
